.circleci/config.yml

```diff
@@ -51,6 +51,9 @@ jobs:
       - run:
           name: run tests
           command: python3 -m unittest
+      - run:
+          name: type check
+          command: mypy --strict-optional --ignore-missing-imports --follow-imports=skip robot/
```

Comment:
Can we set these options in a config file somewhere so editors can also pick them up? (and/or so that just running `mypy` without specific arguments, which is what I expect editors will do, will pick them up)

Reply:
These can be configured in `setup.ini` I think.

Reply:
Indeed, `mypy` reads `setup.cfg`. I've been putting all our lint configuration there as it has the benefits of being a single easy-to-see place where they're all defined as well as allowing for trivial invocations of all the tools. The latter is useful for editor support too.

See also https://github.com/sourcebots/sb-vision/pull/21.